Content.Shared/Administration/SetOutfitEuiState.cs

```diff
@@ -0,0 +1,16 @@
+using Content.Shared.Eui;
+using Robust.Shared.Serialization;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Content.Shared.Administration
+{
+    [Serializable, NetSerializable]
+    public class SetOutfitEuiState : EuiStateBase
+    {
+        public string TargetEntityId;
```

Comment:
Instead of sending an Entity ID as a string, send it as an EntityUid, which has a `ToString()` overload for all your needs!